dyn_em/solve_em.F

```diff
@@ -4632,3 +4657,58 @@ SUBROUTINE solve_em ( grid , config_flags  &
    RETURN
 
 END SUBROUTINE solve_em
+
```

Comment:
@thomasedds 
Thomas,
We do not want to have the source code for a diagnostic as part of the solve_em.F file. The diagnostics properly belong in the physics directory. You might be able to then use the original MSLP code.